504.xml

```diff
@@ -52,123 +52,137 @@ https://www.omaspecworks.org/about/intellectual-property-rights/
 				<MultipleInstances>Single</MultipleInstances>
-				<Mandatory>Mandatory</Mandatory>
+				<Mandatory>Optional</Mandatory>
 				<Type>String</Type>
 				<RangeEnumeration>0..255</RangeEnumeration>
 				<Units></Units>
-				<Description><![CDATA[URI from where the device can download the profile package by an alternative mechanism. As soon the device has received the Profile package URI it performs the download at the next practical opportunity. The URI format is defined in RFC 3986. For example, coaps://example.org/profile is a syntactically valid URI. The URI scheme determines the protocol to be used. For CoAP this endpoint MAY be a LwM2M Server but does not necessarily need to be. A CoAP server implementing block-wise transfer is sufficient as a server hosting a firmware repository and the expectation is that this server merely serves as a separate file server making profile images available to LwM2M Clients. This server can be the future carrier server as well from which IoT devices would like to use the service.]]></Description>
+				<Description><![CDATA[URI, or formatted data containing the URI (e.g. GSMA SGP.22 Activation Code), from where the device can download the profile package by an alternative mechanism. 
+The URI format is defined in RFC 3986. For example, coaps://example.org/profile is a syntactically valid URI. The URI scheme determines the protocol to be used. For CoAP this endpoint MAY be a LwM2M Server
+but does not necessarily need to be. A CoAP server implementing block-wise transfer is sufficient as a server hosting a firmware repository and the expectation is that this server merely serves as a separate
+file server making profile images available to LwM2M Clients. This server can be the future carrier server as well from which IoT devices would like to use the service.]]></Description>
 			</Item>
 			<Item ID="5">
-				<Name>Profile Update</Name>
+				<Name>RSP Update</Name>
 				<Operations>E</Operations>
 				<MultipleInstances>Single</MultipleInstances>
 				<Mandatory>Optional</Mandatory>
-				<Type></Type>
-				<RangeEnumeration></RangeEnumeration>
+				<Type>String</Type>
+				<RangeEnumeration>0..5</RangeEnumeration>
 				<Units></Units>
-				<Description><![CDATA[Updates profile by using the profile package stored in Package, or, by using the profile downloaded from the Package URI. This Resource is only executable when the value of the State Resource is Downloaded.]]></Description>
+				<Description><![CDATA[Executes one or more RSP actions. The actions are given as arguments and in case providing multiple actions comma separation is used, e.g. 0,1,2. The following values are possible:
+0: Download profile
+1: Install profile
+2: Enable profile - for profile identified by resource 26
+3: Disable profile - for profile identified by resource 26
+4: Delete profile - for profile identified by resource 26
+5: Reset memory of current SIM
+6-24: Reserved for future use]]></Description>
```

Comment:
This is a description of the arguments to the executable resource and should be captured appropriately.